Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    time_splitter (1.1.0)
+    time_splitter (1.3.0)
```

Comment:
When I got the file it was at 1.2.0 so I bumped by 1. I have no idea why this is like this now.